internal/util/idlocker.go

```diff
@@ -108,13 +108,10 @@ func (ol *OperationLock) tryAcquire(op operation, volumeID string) error {
 	defer ol.mux.Unlock()
 	switch op {
 	case createOp:
-		// During snapshot create operation there should not be any delete
-		// operation going on for the volume.
-
-		// check any delete operation is going on for given volume ID
-		if _, ok := ol.locks[deleteOp][volumeID]; ok {
```

Comment:
this can cause a problem if someone is using an older snapshotter that does not have the fix for this one. If this is well tested we can mention what version of snapshotter they should be used to avoid this issue.